From 83f933e7ede1b073a371031632c2043a4287df3f Mon Sep 17 00:00:00 2001
From: Bill Yi <byi@google.com>
Date: Wed, 3 Nov 2021 14:28:55 -0700
Subject: [PATCH] Import translations. DO NOT MERGE ANYWHERE

Bug: 193890833
Auto-generated-cl: translation import
Change-Id: I6966abfdfdbf672b27528c1761eb23b95aefe521
(cherry picked from commit e4b62b608847d8f4d1a2f0086e86f49879d58f58)
(cherry picked from commit 038c6705bce9794044bd044c2e4d3cee1c38f229)
Merged-In:I6966abfdfdbf672b27528c1761eb23b95aefe521
---
 res/values-af/strings.xml        | 2 +-
 res/values-am/strings.xml        | 2 +-
 res/values-ar/strings.xml        | 2 +-
 res/values-as/strings.xml        | 2 +-
 res/values-az/strings.xml        | 2 +-
 res/values-b+sr+Latn/strings.xml | 2 +-
 res/values-be/strings.xml        | 2 +-
 res/values-bg/strings.xml        | 2 +-
 res/values-bn/strings.xml        | 2 +-
 res/values-bs/strings.xml        | 2 +-
 res/values-ca/strings.xml        | 2 +-
 res/values-cs/strings.xml        | 2 +-
 res/values-da/strings.xml        | 2 +-
 res/values-de/strings.xml        | 2 +-
 res/values-el/strings.xml        | 2 +-
 res/values-en-rAU/strings.xml    | 2 +-
 res/values-en-rCA/strings.xml    | 2 +-
 res/values-en-rGB/strings.xml    | 2 +-
 res/values-en-rIN/strings.xml    | 2 +-
 res/values-en-rXC/strings.xml    | 2 +-
 res/values-es-rUS/strings.xml    | 2 +-
 res/values-es/strings.xml        | 2 +-
 res/values-et/strings.xml        | 2 +-
 res/values-eu/strings.xml        | 2 +-
 res/values-fa/strings.xml        | 2 +-
 res/values-fi/strings.xml        | 2 +-
 res/values-fr-rCA/strings.xml    | 2 +-
 res/values-fr/strings.xml        | 2 +-
 res/values-gl/strings.xml        | 2 +-
 res/values-gu/strings.xml        | 2 +-
 res/values-hi/strings.xml        | 2 +-
 res/values-hr/strings.xml        | 2 +-
 res/values-hu/strings.xml        | 2 +-
 res/values-hy/strings.xml        | 2 +-
 res/values-in/strings.xml        | 2 +-
 res/values-is/strings.xml        | 2 +-
 res/values-it/strings.xml        | 2 +-
 res/values-iw/strings.xml        | 2 +-
 res/values-ja/strings.xml        | 2 +-
 res/values-ka/strings.xml        | 2 +-
 res/values-kk/strings.xml        | 2 +-
 res/values-km/strings.xml        | 2 +-
 res/values-kn/strings.xml        | 2 +-
 res/values-ko/strings.xml        | 2 +-
 res/values-ky/strings.xml        | 2 +-
 res/values-lo/strings.xml        | 2 +-
 res/values-lt/strings.xml        | 2 +-
 res/values-lv/strings.xml        | 2 +-
 res/values-mk/strings.xml        | 2 +-
 res/values-ml/strings.xml        | 2 +-
 res/values-mn/strings.xml        | 2 +-
 res/values-mr/strings.xml        | 2 +-
 res/values-ms/strings.xml        | 2 +-
 res/values-my/strings.xml        | 2 +-
 res/values-nb/strings.xml        | 2 +-
 res/values-ne/strings.xml        | 2 +-
 res/values-nl/strings.xml        | 2 +-
 res/values-or/strings.xml        | 2 +-
 res/values-pa/strings.xml        | 2 +-
 res/values-pl/strings.xml        | 2 +-
 res/values-pt-rBR/strings.xml    | 2 +-
 res/values-pt-rPT/strings.xml    | 2 +-
 res/values-pt/strings.xml        | 2 +-
 res/values-ro/strings.xml        | 2 +-
 res/values-ru/strings.xml        | 2 +-
 res/values-si/strings.xml        | 2 +-
 res/values-sk/strings.xml        | 2 +-
 res/values-sl/strings.xml        | 2 +-
 res/values-sq/strings.xml        | 2 +-
 res/values-sr/strings.xml        | 2 +-
 res/values-sv/strings.xml        | 2 +-
 res/values-sw/strings.xml        | 2 +-
 res/values-ta/strings.xml        | 2 +-
 res/values-te/strings.xml        | 2 +-
 res/values-th/strings.xml        | 2 +-
 res/values-tl/strings.xml        | 2 +-
 res/values-tr/strings.xml        | 2 +-
 res/values-uk/strings.xml        | 2 +-
 res/values-ur/strings.xml        | 2 +-
 res/values-uz/strings.xml        | 2 +-
 res/values-vi/strings.xml        | 2 +-
 res/values-zh-rCN/strings.xml    | 2 +-
 res/values-zh-rHK/strings.xml    | 2 +-
 res/values-zh-rTW/strings.xml    | 2 +-
 res/values-zu/strings.xml        | 2 +-
 85 files changed, 85 insertions(+), 85 deletions(-)

diff --git a/res/values-af/strings.xml b/res/values-af/strings.xml
index 398fca0e13..73096d719a 100644
--- a/res/values-af/strings.xml
+++ b/res/values-af/strings.xml
@@ -2093,7 +2093,7 @@
     <string name="internal_storage" msgid="999496851424448809">"Interne berging"</string>
     <string name="recompute_size" msgid="1098091228370999128">"Herbereken tans grootte…"</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"Vee programdata uit?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"Alle data van hierdie program sal permanent uitgevee word. Dit sluit alle lêers, instellings, rekeninge, databasisse ens. in."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"Hierdie program se data sal permanent uitgevee word. Dit sluit lêers, instellings, databasisse en ander programdata in."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"OK"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"Kanselleer"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-am/strings.xml b/res/values-am/strings.xml
index b11d87445f..aaac29e803 100644
--- a/res/values-am/strings.xml
+++ b/res/values-am/strings.xml
@@ -2093,7 +2093,7 @@
     <string name="internal_storage" msgid="999496851424448809">"የውስጥ ማከማቻ"</string>
     <string name="recompute_size" msgid="1098091228370999128">"መጠን ድጋሚ በማስላት ላይ..."</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"መተግበሪያ ውሂብ ሰርዝ?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"የዚህ መተግበሪያ ውሂቦች ሁሉ በቋሚነት ይሰረዛሉ።እነዚህም ፋይሎችን፣ ቅንብሮችን፣ መለያዎችን፣ የውሂብ ጎታዎች እና የመሳሰሉትን ያካትታሉ።"</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"የዚህ መተግበሪያ ውሂብ እስከመጨረሻው ይሰረዛል። ይህ ፋይሎችን፣ ቅንብሮችን፣ የውሂብ ጎታን እና ሌላ የመተግበሪያ ውሂብን ያካትታል።"</string>
     <string name="dlg_ok" msgid="1421350367857960997">"እሺ"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"ይቅር"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-ar/strings.xml b/res/values-ar/strings.xml
index a67e049b24..47d6130936 100644
--- a/res/values-ar/strings.xml
+++ b/res/values-ar/strings.xml
@@ -2189,7 +2189,7 @@
     <string name="internal_storage" msgid="999496851424448809">"وحدة التخزين الداخلية"</string>
     <string name="recompute_size" msgid="1098091228370999128">"جارٍ إعادة حساب الحجم..."</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"هل تريد حذف بيانات التطبيق؟"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"سيتم حذف جميع بيانات هذا التطبيق نهائيًا. ويشمل ذلك جميع الملفات والإعدادات والحسابات وقواعد البيانات وما إلى ذلك."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"سيتم حذف بيانات هذا التطبيق نهائيًا، بما في ذلك الملفات والإعدادات وقواعد البيانات وبيانات التطبيق الأخرى."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"حسنًا"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"إلغاء"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-as/strings.xml b/res/values-as/strings.xml
index 0ce62d8fa0..5b65255f94 100644
--- a/res/values-as/strings.xml
+++ b/res/values-as/strings.xml
@@ -2093,7 +2093,7 @@
     <string name="internal_storage" msgid="999496851424448809">"আভ্যন্তৰীণ সঞ্চয়াগাৰ"</string>
     <string name="recompute_size" msgid="1098091228370999128">"এপৰ আকাৰ পুনঃগণনা কৰা হৈছে…"</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"এপ্ ডেটা মচিবনে?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"এই এপটোৰ সকলো ডেটা স্থায়ীভাৱে মচা হ\'ব। সেই ডেটাসমূহৰ ভিতৰত ফাইল ছেটিং, একাউণ্ট, ডেটাবে\'ছ আদি অন্তৰ্ভুক্ত।"</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"এই এপৰ ডেটাখিনি স্থায়ীভাৱে মচা হ’ব। ইয়াত ফাইল, ছেটিং, ডেটাবে’ছ আৰু অন্য এপৰ ডেটা অন্তৰ্ভুক্ত।"</string>
     <string name="dlg_ok" msgid="1421350367857960997">"ঠিক"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"বাতিল কৰক"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-az/strings.xml b/res/values-az/strings.xml
index 7792d74c34..c3fa780370 100644
--- a/res/values-az/strings.xml
+++ b/res/values-az/strings.xml
@@ -2093,7 +2093,7 @@
     <string name="internal_storage" msgid="999496851424448809">"Daxili yaddaş"</string>
     <string name="recompute_size" msgid="1098091228370999128">"Ölçünü yenidən hesablayır..."</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"Tətbiq məlumatları silinsin?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"Bu tətbiq datası həmişəlik silinəcək. Buraya bütün fayllar, ayarlar, hesablar, verilənlər bazası və sairləri aiddir."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"Bu tətbiq datası həmişəlik silinəcək. Buraya fayllar, ayarlar, data bazaları və digər tətbiq datası daxildir."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"OK"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"Ləğv edin"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-b+sr+Latn/strings.xml b/res/values-b+sr+Latn/strings.xml
index 4380983364..e8800ae80d 100644
--- a/res/values-b+sr+Latn/strings.xml
+++ b/res/values-b+sr+Latn/strings.xml
@@ -2117,7 +2117,7 @@
     <string name="internal_storage" msgid="999496851424448809">"Interno skladište"</string>
     <string name="recompute_size" msgid="1098091228370999128">"Ponovno izračunavanje veličine..."</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"Želite li da izbrišete podatke o aplikaciji?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"Svi podaci ove aplikacije biće trajno izbrisani. To obuhvata sve datoteke, podešavanja, naloge, baze podataka itd."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"Podaci ove aplikacije će se trajno izbrisati. To obuhvata fajlove, podešavanja, baze podataka i druge podatke aplikacija."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"Potvrdi"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"Otkaži"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-be/strings.xml b/res/values-be/strings.xml
index 48be9d01ce..71a13253fb 100644
--- a/res/values-be/strings.xml
+++ b/res/values-be/strings.xml
@@ -2143,7 +2143,7 @@
     <string name="internal_storage" msgid="999496851424448809">"Унутраная памяць"</string>
     <string name="recompute_size" msgid="1098091228370999128">"Пераразлік памеру..."</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"Выдаліць даныя праграмы?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"Усе даныя гэтай праграмы будуць назаўсёды выдалены. Гэта датычыцца ўсiх файлаў, налад, уліковых запісаў, баз даных і г. д."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"Даныя гэтай праграмы (файлы, налады, базы даных і г. д.) будуць выдалены назаўсёды."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"ОК"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"Скасаваць"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-bg/strings.xml b/res/values-bg/strings.xml
index 321f0f2d31..0f13d09d71 100644
--- a/res/values-bg/strings.xml
+++ b/res/values-bg/strings.xml
@@ -2095,7 +2095,7 @@
     <string name="internal_storage" msgid="999496851424448809">"Вътр. хранилище"</string>
     <string name="recompute_size" msgid="1098091228370999128">"Размерът се преизчислява..."</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"Да се изтрият ли данните от приложенията?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"Всички данни на това приложение ще бъдат изтрити за постоянно. Това включва всички файлове, настройки, профили и т.н."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"Данните на приложението ще бъдат изтрити за постоянно. Това включва файловете, настройките, базите от данни и други данни от приложението."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"OK"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"Отказ"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-bn/strings.xml b/res/values-bn/strings.xml
index d95ba89590..491a1507d1 100644
--- a/res/values-bn/strings.xml
+++ b/res/values-bn/strings.xml
@@ -2093,7 +2093,7 @@
     <string name="internal_storage" msgid="999496851424448809">"ইন্টারনাল স্টোরেজ"</string>
     <string name="recompute_size" msgid="1098091228370999128">"আকার আবার গণনা করা হচ্ছে..."</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"অ্যাপ ডেটা মুছবেন?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"এই অ্যাপের সমস্ত ডেটা স্থায়ীভাবে মুছে যাবে। সমস্ত ফাইল, সেটিংস, অ্যাকাউন্ট, ডেটাবেস ইত্যাদি সবই।"</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"এই অ্যাপের ডেটা স্থায়ীভাবে মুছে ফেলা হবে। এর মধ্যে ফাইল, সেটিংস, ডেটাবেস এবং অন্য অ্যাপ ডেটা আছে।"</string>
     <string name="dlg_ok" msgid="1421350367857960997">"ঠিক আছে"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"বাতিল"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-bs/strings.xml b/res/values-bs/strings.xml
index f604eb5738..b4e35c2203 100644
--- a/res/values-bs/strings.xml
+++ b/res/values-bs/strings.xml
@@ -2117,7 +2117,7 @@
     <string name="internal_storage" msgid="999496851424448809">"Unutrašnja pohrana"</string>
     <string name="recompute_size" msgid="1098091228370999128">"Ponovno izračunavanje veličine…"</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"Izbrisati podatke aplikacija?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"Svi podaci ove aplikacije će biti trajno izbrisani. To uključuje sve fajlove, postavke, račune, baze podataka itd."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"Ovi podaci aplikacije će se trajno izbrisati. To obuhvata fajlove, postavke, baze podataka i ostale podatke aplikacije."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"Uredu"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"Otkaži"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-ca/strings.xml b/res/values-ca/strings.xml
index a23329ab0d..832ae14db8 100644
--- a/res/values-ca/strings.xml
+++ b/res/values-ca/strings.xml
@@ -2093,7 +2093,7 @@
     <string name="internal_storage" msgid="999496851424448809">"Emmagatzematge intern"</string>
     <string name="recompute_size" msgid="1098091228370999128">"S\'està tornant a calcular la mida..."</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"Vols suprimir les dades de l\'aplicació?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"Se suprimiran totes les dades d\'aquesta aplicació permanentment. Això inclou tots els fitxers, les opcions de configuració, els comptes, les bases de dades, etc."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"Les dades d'aquesta aplicació se suprimiran permanentment. Això inclou fitxers, opcions de configuració, bases de dades i altres dades de l'aplicació."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"D\'acord"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"Cancel·la"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-cs/strings.xml b/res/values-cs/strings.xml
index 6106eb5f94..e3ccef4806 100644
--- a/res/values-cs/strings.xml
+++ b/res/values-cs/strings.xml
@@ -2141,7 +2141,7 @@
     <string name="internal_storage" msgid="999496851424448809">"Interní úložiště"</string>
     <string name="recompute_size" msgid="1098091228370999128">"Přepočítávání velikosti..."</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"Smazat data aplikace?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"Veškerá data (všechny soubory, nastavení, účty, databáze apod.) této aplikace budou trvale vymazána."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"Data této aplikace budou trvale smazána. To zahrnuje soubory, nastavení, databáze a další data aplikace."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"OK"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"Zrušit"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-da/strings.xml b/res/values-da/strings.xml
index 1dd83065a9..3d03f0363e 100644
--- a/res/values-da/strings.xml
+++ b/res/values-da/strings.xml
@@ -2093,7 +2093,7 @@
     <string name="internal_storage" msgid="999496851424448809">"Intern lagerplads"</string>
     <string name="recompute_size" msgid="1098091228370999128">"Beregner størrelse igen..."</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"Vil du slette appdata?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"Alle data i denne app slettes permanent. Det omfatter alle filer, indstillinger, konti, databaser osv."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"Denne apps data slettes permanent. Det omfatter filer, indstillinger, databaser og andre appdata."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"OK"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"Annuller"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-de/strings.xml b/res/values-de/strings.xml
index 549b3500a0..28dbe6199d 100644
--- a/res/values-de/strings.xml
+++ b/res/values-de/strings.xml
@@ -2096,7 +2096,7 @@
     <string name="internal_storage" msgid="999496851424448809">"Interner Speicher"</string>
     <string name="recompute_size" msgid="1098091228370999128">"Größe wird neu berechnet..."</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"App-Daten löschen?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"Alle Daten dieser App werden endgültig gelöscht. Dazu zählen alle Dateien, Einstellungen, Konten, Datenbanken usw."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"Die Daten dieser App werden endgültig gelöscht. Dazu gehören Dateien, Einstellungen, Datenbanken und sonstige App-Daten."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"Ok"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"Abbrechen"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-el/strings.xml b/res/values-el/strings.xml
index 17f13f81b3..c002baba71 100644
--- a/res/values-el/strings.xml
+++ b/res/values-el/strings.xml
@@ -2093,7 +2093,7 @@
     <string name="internal_storage" msgid="999496851424448809">"Εσ. αποθ. χώρος"</string>
     <string name="recompute_size" msgid="1098091228370999128">"Εκ νέου υπολογισμός μεγέθους..."</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"Διαγραφή δεδομένων εφαρμογής;"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"Όλα τα δεδομένα της εφαρμογής θα διαγραφούν μόνιμα. Αυτό περιλαμβάνει όλα τα αρχεία, τις ρυθμίσεις, τους λογαριασμούς, τις βάσεις δεδομένων κ.λπ."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"Τα δεδομένα αυτής της εφαρμογής θα διαγραφούν οριστικά. Σε αυτά περιλαμβάνονται αρχεία, ρυθμίσεις, βάσεις δεδομένων και άλλα δεδομένα εφαρμογών."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"OK"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"Ακύρωση"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-en-rAU/strings.xml b/res/values-en-rAU/strings.xml
index 9bca5d8838..414cea4327 100644
--- a/res/values-en-rAU/strings.xml
+++ b/res/values-en-rAU/strings.xml
@@ -2093,7 +2093,7 @@
     <string name="internal_storage" msgid="999496851424448809">"Internal storage"</string>
     <string name="recompute_size" msgid="1098091228370999128">"Recomputing size…"</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"Delete app data?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"All of this app\'s data will be deleted permanently. This includes all files, settings, accounts, databases, etc."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"This app’s data will be permanently deleted. This includes files, settings, databases and other app data."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"OK"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"Cancel"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-en-rCA/strings.xml b/res/values-en-rCA/strings.xml
index 11a2d7005a..f998aa81bb 100644
--- a/res/values-en-rCA/strings.xml
+++ b/res/values-en-rCA/strings.xml
@@ -2093,7 +2093,7 @@
     <string name="internal_storage" msgid="999496851424448809">"Internal storage"</string>
     <string name="recompute_size" msgid="1098091228370999128">"Recomputing size…"</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"Delete app data?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"All of this app\'s data will be deleted permanently. This includes all files, settings, accounts, databases, etc."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"This app’s data will be permanently deleted. This includes files, settings, databases and other app data."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"OK"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"Cancel"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-en-rGB/strings.xml b/res/values-en-rGB/strings.xml
index 0ee7a2984e..d46d4f9d8a 100644
--- a/res/values-en-rGB/strings.xml
+++ b/res/values-en-rGB/strings.xml
@@ -2093,7 +2093,7 @@
     <string name="internal_storage" msgid="999496851424448809">"Internal storage"</string>
     <string name="recompute_size" msgid="1098091228370999128">"Recomputing size…"</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"Delete app data?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"All of this app\'s data will be deleted permanently. This includes all files, settings, accounts, databases, etc."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"This app’s data will be permanently deleted. This includes files, settings, databases and other app data."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"OK"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"Cancel"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-en-rIN/strings.xml b/res/values-en-rIN/strings.xml
index 16b81492ab..a19e681997 100644
--- a/res/values-en-rIN/strings.xml
+++ b/res/values-en-rIN/strings.xml
@@ -2093,7 +2093,7 @@
     <string name="internal_storage" msgid="999496851424448809">"Internal storage"</string>
     <string name="recompute_size" msgid="1098091228370999128">"Recomputing size…"</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"Delete app data?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"All of this app\'s data will be deleted permanently. This includes all files, settings, accounts, databases, etc."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"This app’s data will be permanently deleted. This includes files, settings, databases and other app data."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"OK"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"Cancel"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-en-rXC/strings.xml b/res/values-en-rXC/strings.xml
index 1f67bc5a7b..b2f7f985f7 100644
--- a/res/values-en-rXC/strings.xml
+++ b/res/values-en-rXC/strings.xml
@@ -2093,7 +2093,7 @@
     <string name="internal_storage" msgid="999496851424448809">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‎‏‎‎‏‎‎‎‏‎‏‏‏‏‎‎‏‏‎‏‏‏‎‏‏‏‏‎‏‏‏‎‏‏‎‏‎‎‎‏‎‏‏‏‎‎‏‏‏‏‏‏‎‏‎‎‎‏‏‎‏‎‎‏‎‎‎‏‎‎‏‎‏‎‎‏‎Internal storage‎‏‎‎‏‎"</string>
     <string name="recompute_size" msgid="1098091228370999128">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‎‏‎‎‏‎‎‎‏‎‏‏‏‏‎‎‏‏‏‏‎‎‏‏‏‏‎‏‎‎‏‏‎‏‎‎‎‎‏‎‎‏‏‏‎‏‏‏‏‏‏‎‎‎‎‎‎‏‏‎‏‎‎‏‎‏‏‏‎‏‎‏‏‎‎‎‎Recomputing size…‎‏‎‎‏‎"</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‎‏‎‎‏‎‎‎‏‎‏‏‏‎‏‎‏‎‏‎‎‎‎‎‎‏‎‎‎‏‎‎‏‏‏‎‎‎‏‏‏‏‏‎‏‎‎‏‎‏‎‎‏‎‏‎‎‏‎‏‎‏‏‎‏‎‏‎‏‏‏‎‏‎‎Delete app data?‎‏‎‎‏‎"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‎‏‎‎‏‎‎‎‏‎‏‏‏‏‏‏‏‏‎‏‏‎‏‎‎‏‏‏‎‏‎‎‏‏‎‏‏‎‏‏‎‏‏‏‎‏‎‏‏‎‏‏‏‎‏‎‎‎‏‎‎‏‎‏‎‎‏‎‏‎‏‎‏‎‏‏‏‎‎‎All this app’s data will be deleted permanently. This includes all files, settings, accounts, databases, etc.‎‏‎‎‏‎"</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‎‏‎‎‏‎‎‎‏‎‏‏‏‏‎‎‏‏‏‏‎‏‎‏‏‏‏‏‎‎‎‎‎‏‏‎‎‏‎‎‏‏‎‎‏‏‎‏‎‎‎‎‎‎‏‏‏‏‎‏‎‏‏‎‏‏‎‎‏‏‎‏‏‏‏‎‎This app’s data will be permanently deleted. This includes files, settings, databases, and other app data.‎‏‎‎‏‎"</string>
     <string name="dlg_ok" msgid="1421350367857960997">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‎‏‎‎‏‎‎‎‏‎‏‏‏‏‎‏‏‎‎‏‏‏‎‏‏‏‎‎‏‏‎‏‎‎‏‏‎‏‎‏‎‎‎‎‏‎‎‎‎‎‎‎‏‏‎‎‎‏‎‏‏‎‏‏‎‏‎‎‎‎‎‏‎‎‏‎‏‎OK‎‏‎‎‏‎"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‎‏‎‎‏‎‎‎‏‎‏‏‏‏‏‏‏‎‎‎‏‏‏‏‎‏‎‏‏‎‎‏‎‏‏‎‏‏‏‏‏‏‎‎‎‏‏‏‏‎‎‎‎‏‎‎‎‏‏‎‏‎‎‎‎‏‎‏‏‎‏‎‏‏‏‎‎‏‏‎Cancel‎‏‎‎‏‎"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-es-rUS/strings.xml b/res/values-es-rUS/strings.xml
index 9f7dbaa75e..1e4949f40d 100644
--- a/res/values-es-rUS/strings.xml
+++ b/res/values-es-rUS/strings.xml
@@ -2095,7 +2095,7 @@
     <string name="internal_storage" msgid="999496851424448809">"Almacenamiento interno"</string>
     <string name="recompute_size" msgid="1098091228370999128">"Recalculando el tamaño…"</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"¿Borrar los datos de la app?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"Se eliminarán de forma permanente todos los datos de esta aplicación, incluidos archivos, ajustes, cuentas, bases de datos, etc."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"Se borrarán de forma permanente los datos de esta app. Se incluyen archivos, parámetros de configuración, bases de datos y otros datos de app."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"Aceptar"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"Cancelar"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-es/strings.xml b/res/values-es/strings.xml
index 5095f40342..7bf847fe63 100644
--- a/res/values-es/strings.xml
+++ b/res/values-es/strings.xml
@@ -2093,7 +2093,7 @@
     <string name="internal_storage" msgid="999496851424448809">"Almacenamiento interno"</string>
     <string name="recompute_size" msgid="1098091228370999128">"Recalculando tamaño…"</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"¿Eliminar datos de la aplicación?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"Todos los datos de esta aplicación se eliminarán de forma permanente. Esto incluye todos los archivos, ajustes, cuentas, bases de datos, etc."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"Los datos de esta aplicación se eliminarán de forma permanente, incluidos los archivos, ajustes, bases de datos y otros datos relacionados con ella."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"Aceptar"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"Cancelar"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-et/strings.xml b/res/values-et/strings.xml
index c04f518f01..612adccad1 100644
--- a/res/values-et/strings.xml
+++ b/res/values-et/strings.xml
@@ -2095,7 +2095,7 @@
     <string name="internal_storage" msgid="999496851424448809">"Sisemälu"</string>
     <string name="recompute_size" msgid="1098091228370999128">"Suuruse ümberarvutamine ..."</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"Kas kustutada rakenduse andmed?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"Rakenduse kõik andmed kustutatakse lõplikult. See hõlmab kõiki faile, seadeid, kontosid, andmebaase jne."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"Selle rakenduse andmed kustutatakse jäädavalt. See hõlmab faile, seadeid, andmebaase ja muid rakenduse andmeid."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"OK"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"Tühista"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-eu/strings.xml b/res/values-eu/strings.xml
index b06e6918d8..8a86629983 100644
--- a/res/values-eu/strings.xml
+++ b/res/values-eu/strings.xml
@@ -2093,7 +2093,7 @@
     <string name="internal_storage" msgid="999496851424448809">"Barneko memoria"</string>
     <string name="recompute_size" msgid="1098091228370999128">"Tamaina berriro kalkulatzen…"</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"Aplikazioko datuak ezabatu nahi?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"Aplikazio honetako datu guztiak betiko ezabatuko dira; besteak beste, fitxategi, ezarpen, kontu, datu-base eta abar guztiak."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"Aplikazio honen datuak betiko ezabatuko dira; besteak beste, fitxategiak, ezarpenak, datu-baseak eta aplikazioko beste batu batzuk."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"Ados"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"Utzi"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-fa/strings.xml b/res/values-fa/strings.xml
index 7a8f8d460b..e387d00e36 100644
--- a/res/values-fa/strings.xml
+++ b/res/values-fa/strings.xml
@@ -2093,7 +2093,7 @@
     <string name="internal_storage" msgid="999496851424448809">"فضای ذخیره‌سازی داخلی"</string>
     <string name="recompute_size" msgid="1098091228370999128">"محاسبه مجدد اندازه…"</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"داده‌های برنامه حذف شود؟"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"همه داده‌های این برنامه به صورت دائمی حذف می‌شود. این شامل تمام فایل‌ها، تنظیمات، حساب‌ها، پایگاه‌های داده و دیگر موارد می‌شود."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"داده‌های این برنامه به‌طور دائم حذف خواهد شد. این شامل فایل‌ها، تنظیمات، پایگاه‌های داده، و دیگر داده‌های برنامه می‌شود."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"تأیید"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"لغو"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-fi/strings.xml b/res/values-fi/strings.xml
index b8de5ad6ed..959aa52d61 100644
--- a/res/values-fi/strings.xml
+++ b/res/values-fi/strings.xml
@@ -2093,7 +2093,7 @@
     <string name="internal_storage" msgid="999496851424448809">"Sisäinen tallennustila"</string>
     <string name="recompute_size" msgid="1098091228370999128">"Lasketaan kokoa uudelleen..."</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"Poistetaanko sovelluksen tiedot?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"Kaikki tämän sovelluksen tiedot poistetaan pysyvästi. Tähän sisältyvät kaikki tiedostot, asetukset, tilit, tietokannat jne."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"Sovelluksen data poistetaan pysyvästi. Tämä koskee tiedostoja, asetuksia, tietokantoja ja muuta sovellusdataa."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"OK"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"Peru"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-fr-rCA/strings.xml b/res/values-fr-rCA/strings.xml
index f912270f72..000af661ba 100644
--- a/res/values-fr-rCA/strings.xml
+++ b/res/values-fr-rCA/strings.xml
@@ -2093,7 +2093,7 @@
     <string name="internal_storage" msgid="999496851424448809">"Mémoire de stockage interne"</string>
     <string name="recompute_size" msgid="1098091228370999128">"Calcul de la taille..."</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"Supprimer les données de l\'application?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"Toutes les données de cette application seront supprimées définitivement, y compris tous les fichiers, les paramètres, les comptes, les bases de données, etc."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"Les données de cette application seront définitivement supprimées. Cela inclut les fichiers, les paramètres, les bases de données et d'autres données de l'application."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"OK"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"Annuler"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-fr/strings.xml b/res/values-fr/strings.xml
index fad2f5c640..bebc758574 100644
--- a/res/values-fr/strings.xml
+++ b/res/values-fr/strings.xml
@@ -2093,7 +2093,7 @@
     <string name="internal_storage" msgid="999496851424448809">"Stockage interne"</string>
     <string name="recompute_size" msgid="1098091228370999128">"Calcul de la taille..."</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"Supprimer les données de l\'application ?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"Toutes les données de cette application (fichiers, paramètres, comptes, bases de données, etc.) seront définitivement supprimées."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"Les données de cette application (fichiers, paramètres, comptes, bases de données, etc.) seront définitivement supprimées."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"OK"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"Annuler"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-gl/strings.xml b/res/values-gl/strings.xml
index 27c5dc56cc..9725d1a8cc 100644
--- a/res/values-gl/strings.xml
+++ b/res/values-gl/strings.xml
@@ -2093,7 +2093,7 @@
     <string name="internal_storage" msgid="999496851424448809">"Almacenamento interno"</string>
     <string name="recompute_size" msgid="1098091228370999128">"Recalculando o tamaño…"</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"Eliminar os datos da aplicación?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"Eliminaranse todos os datos desta aplicación permanentemente. Entre estes inclúense todos os ficheiros, opcións de configuracións, contas, bases de datos etc."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"Os datos desta aplicación (como os ficheiros, a configuración e as bases de datos) eliminaranse de forma permanente."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"Aceptar"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"Cancelar"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-gu/strings.xml b/res/values-gu/strings.xml
index a2eda129db..3cdfca79bd 100644
--- a/res/values-gu/strings.xml
+++ b/res/values-gu/strings.xml
@@ -2093,7 +2093,7 @@
     <string name="internal_storage" msgid="999496851424448809">"આંતરિક સ્ટોરેજ"</string>
     <string name="recompute_size" msgid="1098091228370999128">"કદની ફરીથી ગણના કરી રહ્યું છે…"</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"ઍપનો ડેટા ડિલીટ કરીએ?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"આ તમામ ઍપનો ડેટા કાયમી રીતે ડિલીટ કરવામાં આવશે. આમાં તમામ ફાઇલો, સેટિંગ, એકાઉન્ટ, ડેટાબેઝ વગેરે શામેલ છે."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"આ ઍપના ડેટાને કાયમી રીતે કાઢી નાખવામાં આવશે. આમાં ફાઇલો, સેટિંગ, ડેટાબેઝ અને અન્ય ઍપ ડેટા પણ શામેલ હોય છે."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"ઓકે"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"રદ કરો"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-hi/strings.xml b/res/values-hi/strings.xml
index 384f794954..3e9058b911 100644
--- a/res/values-hi/strings.xml
+++ b/res/values-hi/strings.xml
@@ -2093,7 +2093,7 @@
     <string name="internal_storage" msgid="999496851424448809">"मोबाइल मेमोरी"</string>
     <string name="recompute_size" msgid="1098091228370999128">"आकार फिर से परिकलित कर रहा है…"</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"ऐप्लिकेशन डेटा मिटाना चाहते हैं?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"इस ऐप्लिकेशन का पूरा डेटा हमेशा के लिए मिटा दिया जाएगा. इसमें सभी फ़ाइलें, सेटिंग, खाते, डेटाबेस वगैरह शामिल हैं."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"इस ऐप्लिकेशन का डेटा हमेशा के लिए मिटा दिया जाएगा. इसमें फ़ाइलें, सेटिंग, डेटाबेस, और ऐप्लिकेशन का दूसरा डेटा शामिल है."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"ठीक है"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"रद्द करें"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-hr/strings.xml b/res/values-hr/strings.xml
index bc4b8ee897..e2d9c81ee0 100644
--- a/res/values-hr/strings.xml
+++ b/res/values-hr/strings.xml
@@ -2119,7 +2119,7 @@
     <string name="internal_storage" msgid="999496851424448809">"Unutarnja pohrana"</string>
     <string name="recompute_size" msgid="1098091228370999128">"Ponovni izračun veličine…"</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"Izbrisati podatke aplikacije?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"Svi podaci ove aplikacije trajno će se izbrisati. To obuhvaća sve datoteke, postavke, račune, baze podataka itd."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"Podaci aplikacije trajno će se izbrisati. To uključuje datoteke, postavke, baze podataka i druge podatke aplikacije."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"U redu"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"Odustani"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-hu/strings.xml b/res/values-hu/strings.xml
index 907d379a8c..1e2afc27df 100644
--- a/res/values-hu/strings.xml
+++ b/res/values-hu/strings.xml
@@ -2093,7 +2093,7 @@
     <string name="internal_storage" msgid="999496851424448809">"Belső tárhely"</string>
     <string name="recompute_size" msgid="1098091228370999128">"Méret újraszámítása..."</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"Törli az alkalmazás adatait?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"Az alkalmazás minden adatát véglegesen törölni fogja. Ez magában foglalja az összes fájlt, beállítást, fiókot, adatbázist stb."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"Az alkalmazás adatai véglegesen törlődnek. Így törlődni fognak a fájlok, a beállítások, az adatbázisok és az egyéb alkalmazásadatok is."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"OK"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"Mégse"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-hy/strings.xml b/res/values-hy/strings.xml
index 0b397b8e64..3310712ea4 100644
--- a/res/values-hy/strings.xml
+++ b/res/values-hy/strings.xml
@@ -2093,7 +2093,7 @@
     <string name="internal_storage" msgid="999496851424448809">"Ներքին պահոց"</string>
     <string name="recompute_size" msgid="1098091228370999128">"Չափի վերահաշվարկում…"</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"Ջնջե՞լ հավելվածի տվյալները"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"Այս հավելվածի բոլոր տվյալներն ընդմիշտ կջնջվեն: Այն կներառի բոլոր ֆայլերը, կարգավորումները, հաշիվները, տվյալների շտեմարանները և այլն:"</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"Այս հավելվածի տվյալներն ընդմիշտ կջնջվեն։Դա վերաբերում է ֆայլերին, կարգավորումներին, տվյալների շտեմարաններին և հավելվածի այլ տվյալներին։"</string>
     <string name="dlg_ok" msgid="1421350367857960997">"Հաստատել"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"Չեղարկել"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-in/strings.xml b/res/values-in/strings.xml
index be1a1acd01..95fd4923cd 100644
--- a/res/values-in/strings.xml
+++ b/res/values-in/strings.xml
@@ -2093,7 +2093,7 @@
     <string name="internal_storage" msgid="999496851424448809">"Penyimpanan internal"</string>
     <string name="recompute_size" msgid="1098091228370999128">"Menghitung ulang ukuran..."</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"Hapus data aplikasi?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"Semua data aplikasi ini akan dihapus secara permanen, termasuk semua file, setelan, akun, database, dll."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"Data aplikasi ini akan dihapus secara permanen. Data ini termasuk file, setelan, database, dan data aplikasi lainnya."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"Oke"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"Batal"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-is/strings.xml b/res/values-is/strings.xml
index 3ffbcaa108..cfd733b23d 100644
--- a/res/values-is/strings.xml
+++ b/res/values-is/strings.xml
@@ -2093,7 +2093,7 @@
     <string name="internal_storage" msgid="999496851424448809">"Innbyggð geymsla"</string>
     <string name="recompute_size" msgid="1098091228370999128">"Endurreiknar stærð…"</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"Eyða forritsgögnum?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"Öllum gögnum þessa forrits verður eytt fyrir fullt og allt. Í þessu felast allar skrár, stillingar, reikningar, gagnagrunnar o.s.frv."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"Gögnum þessa forrits verður eytt fyrir fullt og allt. Þetta á við um skrár, stillingar, gagnagrunna og önnur forritsgögn."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"Í lagi"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"Hætta við"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-it/strings.xml b/res/values-it/strings.xml
index 4201449788..fc132ed60c 100644
--- a/res/values-it/strings.xml
+++ b/res/values-it/strings.xml
@@ -2093,7 +2093,7 @@
     <string name="internal_storage" msgid="999496851424448809">"Memoria interna"</string>
     <string name="recompute_size" msgid="1098091228370999128">"Ricalcolo dimensioni…"</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"Eliminare i dati dell\'app?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"Tutti i dati dell\'app verranno eliminati definitivamente, inclusi tutti i file, le impostazioni, gli account, i database e così via."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"I dati di questa app verranno eliminati definitivamente, inclusi file, impostazioni, database e altri dati dell'app."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"OK"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"Annulla"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-iw/strings.xml b/res/values-iw/strings.xml
index 2dbf808ff7..581e690209 100644
--- a/res/values-iw/strings.xml
+++ b/res/values-iw/strings.xml
@@ -2143,7 +2143,7 @@
     <string name="internal_storage" msgid="999496851424448809">"אחסון פנימי"</string>
     <string name="recompute_size" msgid="1098091228370999128">"הגודל מחושב מחדש..."</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"למחוק את נתוני האפליקציה?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"כל נתוני האפליקציה יימחקו לתמיד. הנתונים האלה כוללים את כל הקבצים, ההגדרות, החשבונות, מסדי הנתונים וכדומה."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"נתוני האפליקציה יימחקו באופן סופי. לרבות קבצים, הגדרות, מסדי נתונים ונתונים אחרים מהאפליקציה."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"אישור"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"ביטול"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-ja/strings.xml b/res/values-ja/strings.xml
index e0db9c714d..7a03c44ac4 100644
--- a/res/values-ja/strings.xml
+++ b/res/values-ja/strings.xml
@@ -2093,7 +2093,7 @@
     <string name="internal_storage" msgid="999496851424448809">"内部ストレージ"</string>
     <string name="recompute_size" msgid="1098091228370999128">"サイズを再計算中..."</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"アプリのデータを削除しますか？"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"このアプリのすべてのデータ（ファイル、設定、アカウント、データベースを含む）が完全に削除されます。"</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"このアプリのデータが完全に削除されます。ファイル、設定、データベース、その他のアプリデータが対象になります。"</string>
     <string name="dlg_ok" msgid="1421350367857960997">"OK"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"キャンセル"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-ka/strings.xml b/res/values-ka/strings.xml
index a2fb7721fc..38fad4e737 100644
--- a/res/values-ka/strings.xml
+++ b/res/values-ka/strings.xml
@@ -2093,7 +2093,7 @@
     <string name="internal_storage" msgid="999496851424448809">"შიდა მეხსიერება"</string>
     <string name="recompute_size" msgid="1098091228370999128">"ზომის გადათვლა…"</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"წაიშალოს აპის მონაცემები?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"აპის მონაცემები, მათ შორის ყველა ფაილი, პარამეტრი, ანგარიში, მონაცემთა ბაზა და ა.შ. სამუდამოდ წაიშლება."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"ამ აპის მონაცემები სამუდამოდ წაიშლება. ეს ეხება ფაილებს, პარამეტრებს, მონაცემთა ბაზებსა და აპის სხვა მონაცემებს."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"კარგი"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"გაუქმება"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-kk/strings.xml b/res/values-kk/strings.xml
index 45fb1228af..dc5836bc53 100644
--- a/res/values-kk/strings.xml
+++ b/res/values-kk/strings.xml
@@ -2093,7 +2093,7 @@
     <string name="internal_storage" msgid="999496851424448809">"Ішкі жад"</string>
     <string name="recompute_size" msgid="1098091228370999128">"Өлшемін қайта есептеуде…"</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"Қолданба деректері жойылсын ба?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"Бұл қолданбаның барлық дерегі біржола жойылады. Бұған барлық файл, параметр, аккаунт, дерекқор және т.б. кіреді."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"Бұл қолданбаның дерегі біржола жойылады. Оған файлдар, параметрлер, дерекқорлар және тағы басқа қолданба дерегі кіреді."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"Жарайды"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"Бас тарту"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-km/strings.xml b/res/values-km/strings.xml
index f9ff3691c6..4c47902c14 100644
--- a/res/values-km/strings.xml
+++ b/res/values-km/strings.xml
@@ -2093,7 +2093,7 @@
     <string name="internal_storage" msgid="999496851424448809">"ឧបករណ៍​ខាង​ក្នុង"</string>
     <string name="recompute_size" msgid="1098091228370999128">"កំពុង​គណនា​ទំហំ​ឡើងវិញ…"</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"លុប​ទិន្នន័យ​កម្មវិធី?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"ទិន្នន័យ​របស់​កម្មវិធី​នេះ​ទាំងអស់​នឹង​ត្រូវ​បាន​លុប​ជា​អចិន្ត្រៃយ៍។ ទិន្នន័យទាំងនេះរួមមាន ឯកសារ​ទាំងអស់ ការ​កំណត់ គណនី មូលដ្ឋាន​ទិន្នន័យ ។ល។"</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"ទិន្នន័យ​របស់កម្មវិធីនេះ​នឹងត្រូវបានលុប​ជាអចិន្ត្រៃយ៍។ ទិន្នន័យនេះ​រួមមាន​ឯកសារ ការកំណត់ មូលដ្ឋាន​ទិន្នន័យ និង​ទិន្នន័យកម្មវិធី​ផ្សេងទៀត។"</string>
     <string name="dlg_ok" msgid="1421350367857960997">"យល់​ព្រម​"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"បោះ​បង់​"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-kn/strings.xml b/res/values-kn/strings.xml
index 2e3f5fabb5..89bb43145d 100644
--- a/res/values-kn/strings.xml
+++ b/res/values-kn/strings.xml
@@ -2093,7 +2093,7 @@
     <string name="internal_storage" msgid="999496851424448809">"ಆಂತರಿಕ ಸಂಗ್ರಹಣೆ"</string>
     <string name="recompute_size" msgid="1098091228370999128">"ಗಾತ್ರವನ್ನು ಮರುಗಣನೆ ಮಾಡಲಾಗುತ್ತಿದೆ…"</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"ಆ್ಯಪ್ ಡೇಟಾ ಅಳಿಸಬೇಕೇ?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"ಈ ಆ್ಯಪ್‍‍ನ ಎಲ್ಲಾ ಡೇಟಾವನ್ನು ಶಾಶ್ವತವಾಗಿ ಅಳಿಸಲಾಗುತ್ತದೆ. ಇದರಲ್ಲಿ ಎಲ್ಲಾ ಫೈಲ್‌ಗಳು, ಸೆಟ್ಟಿಂಗ್‌ಗಳು, ಖಾತೆಗಳು, ಡೇಟಾಬೇಸ್‌ಗಳು, ಇತ್ಯಾದಿಗಳು ಸೇರಿರುತ್ತವೆ."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"ಈ ಆ್ಯಪ್‌ನ ಡೇಟಾವನ್ನು ಶಾಶ್ವತವಾಗಿ ಅಳಿಸಲಾಗುವುದು. ಇದು ಫೈಲ್‌ಗಳು, ಸೆಟ್ಟಿಂಗ್‌ಗಳು, ಡೇಟಾಬೇಸ್‌ಗಳು ಮತ್ತು ಇತರ ಆ್ಯಪ್ ಡೇಟಾ ಒಳಗೊಂಡಿದೆ."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"ಸರಿ"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"ರದ್ದುಮಾಡಿ"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-ko/strings.xml b/res/values-ko/strings.xml
index 613d8ecc00..9f4947ce1e 100644
--- a/res/values-ko/strings.xml
+++ b/res/values-ko/strings.xml
@@ -2095,7 +2095,7 @@
     <string name="internal_storage" msgid="999496851424448809">"내부 저장소"</string>
     <string name="recompute_size" msgid="1098091228370999128">"크기 다시 계산 중..."</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"앱 데이터를 삭제하시겠습니까?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"앱의 전체 데이터가 영구적으로 삭제됩니다. 여기에는 모든 파일, 설정, 계정, 데이터베이스 등이 포함됩니다."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"이 앱 데이터가 영구적으로 삭제됩니다. 삭제 대상에는 파일, 설정, 데이터베이스, 기타 앱 데이터가 포함됩니다."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"확인"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"취소"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-ky/strings.xml b/res/values-ky/strings.xml
index 294f5e55dd..a0547b7a6f 100644
--- a/res/values-ky/strings.xml
+++ b/res/values-ky/strings.xml
@@ -2093,7 +2093,7 @@
     <string name="internal_storage" msgid="999496851424448809">"Ички эстутум"</string>
     <string name="recompute_size" msgid="1098091228370999128">"Өлчөмү кайра эсептелүүдө…"</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"Колдонмодогу нерселерди өчүрөсүзбү?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"Бул колдонмодогу файлдар, жөндөөлөр, аккаунттар жана маалымат базасы сыяктуу бардык нерселер биротоло өчүрүлөт."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"Бул колдонмодогу маалымат биротоло жок кылынат. Ага файлдар, жөндөөлөр, дайындар базалары жана колдонмонун башка дайындары кирет."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"Жарайт"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"Жок"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-lo/strings.xml b/res/values-lo/strings.xml
index 6f94b261f4..d68c9aee7c 100644
--- a/res/values-lo/strings.xml
+++ b/res/values-lo/strings.xml
@@ -2093,7 +2093,7 @@
     <string name="internal_storage" msgid="999496851424448809">"ພື້ນທີ່ຈັດເກັບຂໍ້ມູນພາຍໃນ"</string>
     <string name="recompute_size" msgid="1098091228370999128">"ກຳລັງຄຳນວນຂະໜາດຄືນໃໝ່..."</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"ລຶບຂໍ້ມູນແອັບບໍ່?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"ຂໍ້ມູນທັງໝົດຂອງແອັບນີ້ຈະຖືກລຶບອອກຖາວອນ ເຊິ່ງຮວມເຖິງໄຟລ໌, ການຕັ້ງຄ່າ, ບັນຊີ, ຖານຂໍ້ມູນ ແລະ ອື່ນໆທັງໝົດນຳ."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"ຂໍ້ມູນຂອງແອັບນີ້ຈະຖືກລຶບອອກຖາວອນ. ນີ້ຮວມເຖິງໄຟລ໌, ການຕັ້ງຄ່າ, ຖານຂໍ້ມູນ ແລະ ຂໍ້ມູນແອັບອື່ນໆນຳ"</string>
     <string name="dlg_ok" msgid="1421350367857960997">"ຕົກລົງ"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"ຍົກເລີກ"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-lt/strings.xml b/res/values-lt/strings.xml
index 5e48f86145..b3607fecd6 100644
--- a/res/values-lt/strings.xml
+++ b/res/values-lt/strings.xml
@@ -2141,7 +2141,7 @@
     <string name="internal_storage" msgid="999496851424448809">"Vidinė atmint."</string>
     <string name="recompute_size" msgid="1098091228370999128">"Iš naujo apskaičiuojamas dydis..."</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"Ištrinti programos duomenis?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"Visi šios programos duomenys bus visam laikui ištrinti. Tai taikoma visiems failams, nustatymams, paskyroms, duomenims ir kt."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"Šios programos duomenys bus visam laikui ištrinti. Įtraukiami failai, nustatymai, duomenų bazės ir kiti programos duomenys."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"Gerai"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"Atšaukti"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-lv/strings.xml b/res/values-lv/strings.xml
index d1c1f64b13..088b640e29 100644
--- a/res/values-lv/strings.xml
+++ b/res/values-lv/strings.xml
@@ -2117,7 +2117,7 @@
     <string name="internal_storage" msgid="999496851424448809">"Iekšējā krātuve"</string>
     <string name="recompute_size" msgid="1098091228370999128">"Notiek izmēra pārrēķināšana..."</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"Vai dzēst lietotnes datus?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"Tiks neatgriezeniski dzēsti visi šīs lietotnes dati, tostarp visi faili, iestatījumi, konti, datu bāzes un citi vienumi."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"Tiks neatgriezeniski dzēsti šīs lietotnes dati, tostarp faili, iestatījumi, datu bāzes un citi lietotnes dati."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"Labi"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"Atcelt"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-mk/strings.xml b/res/values-mk/strings.xml
index 80539845b5..816769370a 100644
--- a/res/values-mk/strings.xml
+++ b/res/values-mk/strings.xml
@@ -2095,7 +2095,7 @@
     <string name="internal_storage" msgid="999496851424448809">"Внатрешен капацитет"</string>
     <string name="recompute_size" msgid="1098091228370999128">"Повторно пресметување големина..."</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"Да се избришат податоците на апликацијата?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"Сите податоци на оваа апликација ќе бидат трајно избришани. Ова ги опфаќа сите датотеки, поставки, сметки, бази на податоци итн."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"Податоците од апликацијава трајно ќе се избришат. Ова вклучува датотеки, поставки, бази на податоци и други податоци од апликацијата."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"Во ред"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"Откажи"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-ml/strings.xml b/res/values-ml/strings.xml
index 1c6c3c9037..f96e44cee1 100644
--- a/res/values-ml/strings.xml
+++ b/res/values-ml/strings.xml
@@ -2093,7 +2093,7 @@
     <string name="internal_storage" msgid="999496851424448809">"ആന്തരിക സ്റ്റോറേജ്"</string>
     <string name="recompute_size" msgid="1098091228370999128">"വലുപ്പം വീണ്ടും കണക്കാക്കുന്നു..."</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"ആപ്പ് ഡാറ്റ ഇല്ലാതാക്കണോ?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"ഈ ആപ്പിന്‍റെ എല്ലാ ഡാറ്റയും ശാശ്വതമായി ഇല്ലാതാക്കും. ഫയലുകൾ, ക്രമീകരണങ്ങൾ, അക്കൗണ്ടുകൾ, ഡാറ്റാബേസുകൾ തുടങ്ങിയവയെല്ലാം ഇതിൽ ഉൾപ്പെടും."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"ഈ ആപ്പിന്റെ ഡാറ്റ ശാശ്വതമായി ഇല്ലാതാക്കും. ഇതിൽ ഫയലുകളും ക്രമീകരണവും ഡാറ്റാബേസുകളും മറ്റ് ആപ്പ് ഡാറ്റയും ഉൾപ്പെടുന്നു."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"ശരി"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"റദ്ദാക്കുക"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-mn/strings.xml b/res/values-mn/strings.xml
index 553f897358..dcf8544c8d 100644
--- a/res/values-mn/strings.xml
+++ b/res/values-mn/strings.xml
@@ -2093,7 +2093,7 @@
     <string name="internal_storage" msgid="999496851424448809">"Дотоод сан"</string>
     <string name="recompute_size" msgid="1098091228370999128">"Хэмжээг дахин тооцоолж байна…"</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"Апп өгөгдлийг устгах уу?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"Энэ апп-н бүх өгөгдөл бүрмөсөн устах болно. Үүнд бүх файл, тохиргоо, бүртгэл, өгөгдөл гэх мэт орно."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"Энэ аппын өгөгдлийг бүрмөсөн устгана. Үүнд файл, тохиргоо, өгөгдлийн бааз болон бусад аппын өгөгдөл багтана."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"OK"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"Цуцлах"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-mr/strings.xml b/res/values-mr/strings.xml
index 89ab80c75d..3b9f320812 100644
--- a/res/values-mr/strings.xml
+++ b/res/values-mr/strings.xml
@@ -2094,7 +2094,7 @@
     <string name="internal_storage" msgid="999496851424448809">"अंतर्गत स्टोरेज"</string>
     <string name="recompute_size" msgid="1098091228370999128">"आकाराचे पुनर्संगणन करत आहे…"</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"अ‍ॅप डेटा हटवायचा?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"या अ‍ॅपचा सर्व डेटा कायमचा हटवला जाईल. यात सर्व फाइल, सेटिंग्ज, खाती, डेटाबेस इ. समाविष्ट असेल."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"या ॲपचा डेटा कायमचा हटवला जाईल. यामध्ये फाइल, सेटिंग्ज, डेटाबेस आणि इतर अ‍ॅप डेटा यांचा समावेश आहे."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"ठीक"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"रद्द करा"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-ms/strings.xml b/res/values-ms/strings.xml
index 56634361bc..bedb34e4f1 100644
--- a/res/values-ms/strings.xml
+++ b/res/values-ms/strings.xml
@@ -2093,7 +2093,7 @@
     <string name="internal_storage" msgid="999496851424448809">"Storan dalaman"</string>
     <string name="recompute_size" msgid="1098091228370999128">"Mengira semula saiz..."</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"Padamkan data aplikasi?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"Semua data aplikasi ini akan dipadamkan secara kekal. Ini termasuk semua fail, tetapan, akaun, pangkalan data, dll."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"Data apl ini akan dipadamkan secara kekal. Kandungan ini termasuk fail, tetapan, pangkalan data dan data apl yang lain."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"OK"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"Batal"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-my/strings.xml b/res/values-my/strings.xml
index 2273f88e58..3d48d63f35 100644
--- a/res/values-my/strings.xml
+++ b/res/values-my/strings.xml
@@ -2093,7 +2093,7 @@
     <string name="internal_storage" msgid="999496851424448809">"စက်တွင်း သိုလှောင်ထားမှု"</string>
     <string name="recompute_size" msgid="1098091228370999128">"အရွယ်အစားကို ပြန်လည်တွက်ချက်နေသည်"</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"အက်ပ်ဒေတာ ဖျက်မလား။"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"ဤအက်ပ်၏ ဒေတာအားလုံးကို အပြီးဖျက်ပါမည်။ ဖိုင်၊ ဆက်တင်၊ အကောင့်၊ ဒေတာဘေ့စ်စသည်တို့ အားလုံးပါဝင်သည်။"</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"ဤအက်ပ်၏ ဒေတာကို အပြီးဖျက်ပါမည်။ ၎င်းတွင် ဖိုင်၊ ဆက်တင်၊ ဒေတာဘေ့စ်နှင့် အခြားအက်ပ် ဒေတာများ ပါဝင်သည်။"</string>
     <string name="dlg_ok" msgid="1421350367857960997">"OK"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"မလုပ်တော့"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-nb/strings.xml b/res/values-nb/strings.xml
index 8d51ec6fb9..15314c9c6f 100644
--- a/res/values-nb/strings.xml
+++ b/res/values-nb/strings.xml
@@ -2093,7 +2093,7 @@
     <string name="internal_storage" msgid="999496851424448809">"Intern lagring"</string>
     <string name="recompute_size" msgid="1098091228370999128">"Beregner størrelse på nytt…"</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"Vil du slette appdataene?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"Alle data i appen slettes permanent. Dette omfatter alle filer, innstillinger, kontoer, databaser osv."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"Dataene i denne appen slettes permanent. Dette inkluderer filer, innstillinger, databaser og andre appdata."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"OK"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"Avbryt"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-ne/strings.xml b/res/values-ne/strings.xml
index 9976ff52a3..3d359067bc 100644
--- a/res/values-ne/strings.xml
+++ b/res/values-ne/strings.xml
@@ -2093,7 +2093,7 @@
     <string name="internal_storage" msgid="999496851424448809">"आन्तरिक भण्डारण"</string>
     <string name="recompute_size" msgid="1098091228370999128">"आकार पुनःगणना गर्दै ..."</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"एपको डेटा मेटाउन चाहनुहुन्छ?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"यस एपका सम्पूर्ण डेटाहरू स्थायी रूपमा मेटाइने छ। यसमा सम्पूर्ण फाइल, सेटिङ, खाताहरू, डेटाबेस आदि पर्दछन्।"</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"यो एपका डेटा सदाका लागि मेटाइने छ। यसमा फाइल, सेटिङ, डेटाबेस र एपका अन्य डेटा पर्छन्।"</string>
     <string name="dlg_ok" msgid="1421350367857960997">"ठिक छ"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"रद्द गर्नुहोस्"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-nl/strings.xml b/res/values-nl/strings.xml
index a63c1f7db1..2df67bbeb9 100644
--- a/res/values-nl/strings.xml
+++ b/res/values-nl/strings.xml
@@ -2093,7 +2093,7 @@
     <string name="internal_storage" msgid="999496851424448809">"Interne opslag"</string>
     <string name="recompute_size" msgid="1098091228370999128">"Grootte opnieuw berekenen..."</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"Gegevens van app verwijderen?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"Alle gegevens van deze app worden permanent verwijderd. Dit omvat alle bestanden, instellingen, accounts, databases, enzovoort."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"De gegevens van deze app worden definitief verwijderd. Dit omvat bestanden, instellingen, databases en andere app-gegevens."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"OK"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"Annuleren"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-or/strings.xml b/res/values-or/strings.xml
index 772e734ea4..56070e3b20 100644
--- a/res/values-or/strings.xml
+++ b/res/values-or/strings.xml
@@ -2093,7 +2093,7 @@
     <string name="internal_storage" msgid="999496851424448809">"ଇଣ୍ଟର୍ନଲ୍‌ ଷ୍ଟୋରେଜ୍‌"</string>
     <string name="recompute_size" msgid="1098091228370999128">"ଆକାରକୁ ପୁଣି ଗଣୁଛି…"</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"ଆପ୍‌ର ଡାଟା ଡିଲିଟ୍‌ କରିବେ?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"ଏହି ଆପ୍‌ର ସମସ୍ତ ଡାଟା ସ୍ଥାୟୀ ଭାବେ ଡିଲିଟ୍‌ ହୋଇଯିବ। ଏଥିରେ ରହିଛି, ସମସ୍ତ ଫାଇଲ୍‌, ସେଟିଙ୍ଗ, ଆକାଉଣ୍ଟ, ଡାଟାବେସ୍‌ ଆଦି"</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"ଏହି ଆପର ଡାଟା ସ୍ଥାୟୀ ଭାବେ ଡିଲିଟ ହୋଇଯିବ। ଏଥିରେ ଫାଇଲ, ସେଟିଂସ, ଡାଟାବେସ ଏବଂ ଅନ୍ୟ ଆପ ଡାଟା ଅନ୍ତର୍ଭୁକ୍ତ।"</string>
     <string name="dlg_ok" msgid="1421350367857960997">"ଠିକ୍‌ ଅଛି"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"ବାତିଲ କରନ୍ତୁ"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-pa/strings.xml b/res/values-pa/strings.xml
index 8c558ae818..2afd8d12c5 100644
--- a/res/values-pa/strings.xml
+++ b/res/values-pa/strings.xml
@@ -2093,7 +2093,7 @@
     <string name="internal_storage" msgid="999496851424448809">"ਅੰਦਰੂਨੀ ਸਟੋਰੇਜ"</string>
     <string name="recompute_size" msgid="1098091228370999128">"ਅਕਾਰ ਰੀਕੰਪਿਊਟ ਕਰ ਰਿਹਾ ਹੈ..."</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"ਕੀ ਐਪ ਡਾਟਾ ਮਿਟਾਉਣਾ ਹੈ?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"ਇਸ ਐਪ ਦਾ ਸਾਰਾ ਡਾਟਾ ਸਥਾਈ ਤੌਰ ਤੇ ਮਿਟਾਇਆ ਜਾਏਗਾ। ਇਸ ਵਿੱਚ ਸਾਰੀਆਂ ਫ਼ਾਈਲਾਂ, ਸੈਟਿੰਗਾਂ, ਖਾਤੇ, ਡਾਟਾਬੇਸ ਆਦਿ ਸ਼ਾਮਲ ਹਨ।"</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"ਇਸ ਐਪ ਦਾ ਡਾਟਾ ਪੱਕੇ ਤੌਰ 'ਤੇ ਮਿਟਾ ਦਿੱਤਾ ਜਾਵੇਗਾ। ਇਸ ਵਿੱਚ ਫ਼ਾਈਲਾਂ, ਸੈਟਿੰਗਾਂ, ਡਾਟਾਬੇਸ ਅਤੇ ਹੋਰ ਐਪ ਡਾਟਾ ਸ਼ਾਮਲ ਹੈ।"</string>
     <string name="dlg_ok" msgid="1421350367857960997">"ਠੀਕ ਹੈ"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"ਰੱਦ ਕਰੋ"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-pl/strings.xml b/res/values-pl/strings.xml
index a7f30747c1..c7c15d91f9 100644
--- a/res/values-pl/strings.xml
+++ b/res/values-pl/strings.xml
@@ -2141,7 +2141,7 @@
     <string name="internal_storage" msgid="999496851424448809">"Pamięć wewnętrzna"</string>
     <string name="recompute_size" msgid="1098091228370999128">"Przeliczanie rozmiaru..."</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"Usunąć dane aplikacji?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"Wszystkie dane tej aplikacji zostaną trwale usunięte. Dotyczy to wszystkich plików, ustawień, kont, baz danych itd."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"Dane tej aplikacji zostaną trwale usunięte. Dotyczy to plików, ustawień, baz danych i innych danych aplikacji."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"OK"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"Anuluj"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-pt-rBR/strings.xml b/res/values-pt-rBR/strings.xml
index 6c2034c90e..e76fbe5854 100644
--- a/res/values-pt-rBR/strings.xml
+++ b/res/values-pt-rBR/strings.xml
@@ -2095,7 +2095,7 @@
     <string name="internal_storage" msgid="999496851424448809">"Espaço interno"</string>
     <string name="recompute_size" msgid="1098091228370999128">"Recalculando o tamanho…"</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"Excluir os dados do app?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"Todos os dados deste app serão excluídos permanentemente. Isso inclui todos os arquivos, configurações, contas, bancos de dados e outros."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"Os dados do app serão excluídos permanentemente, incluindo arquivos, configurações, bancos de dados, entre outros."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"OK"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"Cancelar"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-pt-rPT/strings.xml b/res/values-pt-rPT/strings.xml
index b7705ec653..7bc69a123e 100644
--- a/res/values-pt-rPT/strings.xml
+++ b/res/values-pt-rPT/strings.xml
@@ -2093,7 +2093,7 @@
     <string name="internal_storage" msgid="999496851424448809">"Armaz. interno"</string>
     <string name="recompute_size" msgid="1098091228370999128">"A recalcular tamanho..."</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"Eliminar dados da app?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"Todos os dados desta aplicação serão eliminados permanentemente. Isto inclui todos os ficheiros, definições, contas, bases de dados, etc."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"Os dados desta app serão permanentemente eliminados. Estão incluídos ficheiros, definições, bases de dados e outros dados de apps."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"OK"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"Cancelar"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-pt/strings.xml b/res/values-pt/strings.xml
index 6c2034c90e..e76fbe5854 100644
--- a/res/values-pt/strings.xml
+++ b/res/values-pt/strings.xml
@@ -2095,7 +2095,7 @@
     <string name="internal_storage" msgid="999496851424448809">"Espaço interno"</string>
     <string name="recompute_size" msgid="1098091228370999128">"Recalculando o tamanho…"</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"Excluir os dados do app?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"Todos os dados deste app serão excluídos permanentemente. Isso inclui todos os arquivos, configurações, contas, bancos de dados e outros."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"Os dados do app serão excluídos permanentemente, incluindo arquivos, configurações, bancos de dados, entre outros."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"OK"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"Cancelar"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-ro/strings.xml b/res/values-ro/strings.xml
index 15c1978eb2..5b8dc3bcdd 100644
--- a/res/values-ro/strings.xml
+++ b/res/values-ro/strings.xml
@@ -2119,7 +2119,7 @@
     <string name="internal_storage" msgid="999496851424448809">"Stocare internă"</string>
     <string name="recompute_size" msgid="1098091228370999128">"Se recalculează dimensiunea..."</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"Ștergeți datele aplicației?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"Toate datele acestei aplicații vor fi șterse definitiv. Aici sunt incluse toate fișierele, setările, conturile, bazele de date etc."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"Datele acestei aplicații vor fi șterse definitiv. Aici sunt incluse fișierele, setările, bazele de date și alte date ale aplicației."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"OK"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"Anulați"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-ru/strings.xml b/res/values-ru/strings.xml
index dc29c406e4..7b3212b3cc 100644
--- a/res/values-ru/strings.xml
+++ b/res/values-ru/strings.xml
@@ -2141,7 +2141,7 @@
     <string name="internal_storage" msgid="999496851424448809">"Внутр. накопитель"</string>
     <string name="recompute_size" msgid="1098091228370999128">"Повторное вычисление размера..."</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"Удалить данные приложения?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"Все данные этого приложения (включая файлы, базы данных, настройки, аккаунты и т. д.) будут удалены без возможности восстановления."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"Данные этого приложения (включая файлы, базы данных, настройки и т. д.) будут удалены без возможности восстановления."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"ОК"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"Отмена"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-si/strings.xml b/res/values-si/strings.xml
index 020f65f407..d005bdf6aa 100644
--- a/res/values-si/strings.xml
+++ b/res/values-si/strings.xml
@@ -2093,7 +2093,7 @@
     <string name="internal_storage" msgid="999496851424448809">"අභ්‍යන්තර ආචයනය"</string>
     <string name="recompute_size" msgid="1098091228370999128">"විශාලත්වය නැවත ගණනය කරමින්…"</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"යෙදුම් දත්ත මකන්නද?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"මෙම යෙදුමේ සියලු දත්ත ස්ථිරව මකා දමනු ඇත. මෙයට සියලු ගොනු, සැකසීම්, ගිණුම්, දත්ත සමූහ සහ තවත් අඩංගුය."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"මෙම යෙදුමේ දත්ත ස්ථිරවම මකනු ඇත. මෙයට ගොනු, සැකසීම්, දත්ත සමුදායන් සහ වෙනත් යෙදුම් දත්ත ඇතුළත් වේ."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"හරි"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"අවලංගු කරන්න"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-sk/strings.xml b/res/values-sk/strings.xml
index 94c4aa1f36..0dec38adb4 100644
--- a/res/values-sk/strings.xml
+++ b/res/values-sk/strings.xml
@@ -2143,7 +2143,7 @@
     <string name="internal_storage" msgid="999496851424448809">"Interný ukl. priestor"</string>
     <string name="recompute_size" msgid="1098091228370999128">"Prebieha prepočítanie veľkosti..."</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"Odstrániť dáta aplikácie?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"Všetky dáta tejto aplikácie vrátane súborov, nastavení, účtov, databáz atď. budú navždy odstránené."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"Dáta tejto aplikácie budú natrvalo odstránené. Zahrnujú súbory, nastavenia, databázy a ďalšie dáta aplikácie."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"OK"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"Zrušiť"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-sl/strings.xml b/res/values-sl/strings.xml
index 0da0904045..0751036386 100644
--- a/res/values-sl/strings.xml
+++ b/res/values-sl/strings.xml
@@ -2143,7 +2143,7 @@
     <string name="internal_storage" msgid="999496851424448809">"Notranja shramba"</string>
     <string name="recompute_size" msgid="1098091228370999128">"Preračunavanje velikosti ..."</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"Želite izbrisati podatke aplikacije?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"Vsi podatki te aplikacije bodo trajno izbrisani. To med drugim vključuje vse datoteke, nastavitve, račune in podatkovne zbirke."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"Podatki te aplikacije bodo trajno izbrisani. To vključuje datoteke, nastavitve, zbirke podatkov in druge podatke aplikacije."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"V redu"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"Prekliči"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-sq/strings.xml b/res/values-sq/strings.xml
index 50b2777d5a..9e32889aa3 100644
--- a/res/values-sq/strings.xml
+++ b/res/values-sq/strings.xml
@@ -2093,7 +2093,7 @@
     <string name="internal_storage" msgid="999496851424448809">"Hapësira e brendshme ruajtëse"</string>
     <string name="recompute_size" msgid="1098091228370999128">"Po llogarit përsëri madhësinë…"</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"Të fshihen të dhënat e aplikacionit?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"Të gjitha të dhënat e këtij aplikacioni do të fshihen përgjithmonë. Kjo përfshin të gjithë skedarët, cilësimet, llogaritë, bazat e të dhënave etj."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"Të dhënat e këtij aplikacioni do të fshihen përgjithmonë. Këtu përfshihen skedarët, cilësimet, bazat e të dhënave dhe të dhëna të tjera të aplikacionit."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"Në rregull"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"Anulo"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-sr/strings.xml b/res/values-sr/strings.xml
index 153f83477c..b79cc59671 100644
--- a/res/values-sr/strings.xml
+++ b/res/values-sr/strings.xml
@@ -2117,7 +2117,7 @@
     <string name="internal_storage" msgid="999496851424448809">"Интерно складиште"</string>
     <string name="recompute_size" msgid="1098091228370999128">"Поновно израчунавање величине..."</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"Желите ли да избришете податке о апликацији?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"Сви подаци ове апликације биће трајно избрисани. То обухвата све датотеке, подешавања, налоге, базе података итд."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"Подаци ове апликације ће се трајно избрисати. То обухвата фајлове, подешавања, базе података и друге податке апликација."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"Потврди"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"Откажи"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-sv/strings.xml b/res/values-sv/strings.xml
index 571e017209..26deb01a1d 100644
--- a/res/values-sv/strings.xml
+++ b/res/values-sv/strings.xml
@@ -2093,7 +2093,7 @@
     <string name="internal_storage" msgid="999496851424448809">"Intern lagring"</string>
     <string name="recompute_size" msgid="1098091228370999128">"Räknar om storlek…"</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"Vill du ta bort appdata?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"Appens samtliga data tas bort permanent. Det inkluderar alla filer, inställningar, konton, databaser och så vidare."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"Appens data raderas permanent. Detta omfattar filer, inställningar, databaser och annan appdata."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"OK"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"Avbryt"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-sw/strings.xml b/res/values-sw/strings.xml
index a8ff6b430b..7489038043 100644
--- a/res/values-sw/strings.xml
+++ b/res/values-sw/strings.xml
@@ -2093,7 +2093,7 @@
     <string name="internal_storage" msgid="999496851424448809">"Hifadhi ya ndani"</string>
     <string name="recompute_size" msgid="1098091228370999128">"Kiwango cha kukadiria upya"</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"Ungependa kufuta data ya programu?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"Data yote ya programu hii itafutwa kabisa. Hii inajumuisha faili, mipangilio, akaunti, hifadhidata zote na mengineyo."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"Data iliyo kwenye programu hii itafutwa kabisa. Hii ni pamoja na faili, mipangilio, hifadhidata na data nyingine ya programu."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"Sawa"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"Ghairi"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-ta/strings.xml b/res/values-ta/strings.xml
index cab378a82d..a2800eb153 100644
--- a/res/values-ta/strings.xml
+++ b/res/values-ta/strings.xml
@@ -2093,7 +2093,7 @@
     <string name="internal_storage" msgid="999496851424448809">"அகச் சேமிப்பிடம்"</string>
     <string name="recompute_size" msgid="1098091228370999128">"அளவை மீண்டும் கணக்கிடுகிறது…"</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"ஆப்ஸ் டேட்டாவை நீக்கவா?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"ஆப்ஸின் எல்லா தகவலும் நிரந்தரமாக நீக்கப்படும். இதில் எல்லா ஃபைல்களும், அமைப்புகளும், கணக்குகளும், தரவுத்தளங்களும், மேலும் பலவும் அடங்கும்."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"இந்த ஆப்ஸின் தரவு நிரந்தரமாக நீக்கப்படும். ஃபைல்கள், அமைப்புகள், தரவுத்தளங்கள், பிற ஆப்ஸ் தரவு ஆகியவை இதிலடங்கும்."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"சரி"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"ரத்துசெய்"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-te/strings.xml b/res/values-te/strings.xml
index 947da9a0b2..69a9bdcb4d 100644
--- a/res/values-te/strings.xml
+++ b/res/values-te/strings.xml
@@ -2093,7 +2093,7 @@
     <string name="internal_storage" msgid="999496851424448809">"అంతర్గత నిల్వ"</string>
     <string name="recompute_size" msgid="1098091228370999128">"పరిమాణాన్ని మళ్లీ గణిస్తోంది…"</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"యాప్ డేటాను తొలగించాలా?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"ఈ యాప్ డేటా మొత్తం శాశ్వతంగా తొలగించబడుతుంది. ఇందులో అన్ని ఫైళ్లు, సెట్టింగ్‌లు, ఖాతాలు, డేటాబేస్‌లు మొ. ఉన్నాయి."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"ఈ యాప్ డేటా శాశ్వతంగా తొలగించబడుతుంది. ఇందులో ఫైల్స్, సెట్టింగ్‌లు, డేటాబేస్‌లు, ఇతర యాప్ డేటా ఉంటాయి."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"సరే"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"రద్దు చేయి"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-th/strings.xml b/res/values-th/strings.xml
index c1abb2e32c..81356a56e0 100644
--- a/res/values-th/strings.xml
+++ b/res/values-th/strings.xml
@@ -2093,7 +2093,7 @@
     <string name="internal_storage" msgid="999496851424448809">"ที่เก็บข้อมูลภายใน"</string>
     <string name="recompute_size" msgid="1098091228370999128">"กำลังคำนวณขนาดใหม่..."</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"ลบข้อมูลในแอปหรือไม่"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"ระบบจะลบข้อมูลทั้งหมดของแอปนี้โดยถาวร ซึ่งรวมถึงไฟล์ การตั้งค่า บัญชี ฐานข้อมูล ฯลฯ"</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"ระบบจะลบข้อมูลของแอปนี้อย่างถาวร ซึ่งรวมถึงไฟล์ การตั้งค่า ฐานข้อมูล และข้อมูลแอปอื่นๆ"</string>
     <string name="dlg_ok" msgid="1421350367857960997">"ตกลง"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"ยกเลิก"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-tl/strings.xml b/res/values-tl/strings.xml
index a687e45737..c59e9f4c29 100644
--- a/res/values-tl/strings.xml
+++ b/res/values-tl/strings.xml
@@ -2093,7 +2093,7 @@
     <string name="internal_storage" msgid="999496851424448809">"Panloob na storage"</string>
     <string name="recompute_size" msgid="1098091228370999128">"Muling kino-compute ang laki…"</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"I-delete ang data ng app?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"Permanenteng made-delete ang lahat ng data ng app na ito. Kabilang dito ang lahat ng file, setting, account, database, atbp."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"Permanenteng made-delete ang data ng app na ito. Kasama rito ang mga file, setting, database, at iba pang data ng app."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"OK"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"Kanselahin"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-tr/strings.xml b/res/values-tr/strings.xml
index cc51b8ec10..b41bec2d6c 100644
--- a/res/values-tr/strings.xml
+++ b/res/values-tr/strings.xml
@@ -2093,7 +2093,7 @@
     <string name="internal_storage" msgid="999496851424448809">"Dahili Bellek"</string>
     <string name="recompute_size" msgid="1098091228370999128">"Boyut yeniden hesaplanıyor…"</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"Uygulama verileri silinsin mi?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"Bu uygulamanın tüm verileri kalıcı olarak silinecek. Bu veriler arasında tüm dosyalar, ayarlar, hesaplar, veritabanları vb. yer alıyor."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"Bu uygulamanın verileri kalıcı olarak silinecek. Buna dosyalar, ayarlar, veritabanları ve diğer uygulama verileri de dahildir."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"Tamam"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"İptal"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-uk/strings.xml b/res/values-uk/strings.xml
index dc6a9641a3..2424b098cf 100644
--- a/res/values-uk/strings.xml
+++ b/res/values-uk/strings.xml
@@ -2141,7 +2141,7 @@
     <string name="internal_storage" msgid="999496851424448809">"Внутр. пам\'ять"</string>
     <string name="recompute_size" msgid="1098091228370999128">"Повт. обчисл. розм…"</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"Видалити дані додатка?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"Усі дані цього додатка буде видалено назавжди: файли, налаштування, облікові записи, бази даних тощо."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"Дані цього додатка буде назавжди видалено. До них належать файли, налаштування, бази даних тощо."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"OK"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"Скасувати"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-ur/strings.xml b/res/values-ur/strings.xml
index 8f9b37010e..cf9791788f 100644
--- a/res/values-ur/strings.xml
+++ b/res/values-ur/strings.xml
@@ -2093,7 +2093,7 @@
     <string name="internal_storage" msgid="999496851424448809">"داخلی اسٹوریج"</string>
     <string name="recompute_size" msgid="1098091228370999128">"سائز کو دوبارہ شمار کر رہا ہے…"</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"ایپ کا ڈیٹا حذف کریں؟"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"اس ایپ کا سبھی ڈیٹا مستقل طور پر حذف ہو جائے گا۔ اس میں سبھی فائلیں، ترتیبات، اکاؤنٹس، ڈیٹا بیسز وغیرہ شامل ہیں۔"</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"اس ایپ کا ڈیٹا مستقل طور پر حذف کر دیا جائے گا۔ اس میں فائلز، ترتیبات، ڈیٹا بیس اور دیگر ایپ کا ڈیٹا شامل ہے۔"</string>
     <string name="dlg_ok" msgid="1421350367857960997">"ٹھیک ہے"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"منسوخ کریں"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-uz/strings.xml b/res/values-uz/strings.xml
index f039831960..b09da08890 100644
--- a/res/values-uz/strings.xml
+++ b/res/values-uz/strings.xml
@@ -2095,7 +2095,7 @@
     <string name="internal_storage" msgid="999496851424448809">"Ichki xotira"</string>
     <string name="recompute_size" msgid="1098091228370999128">"Xajm qayta hisoblanmoqda…"</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"Bu ilova maʼlumotlari oʻchirilsinmi?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"Ushbu ilovaning barcha ma’lumotlari, jumladan, fayllar, ma’lumotlar bazalari, sozlamalar, hisoblar va boshqa narsalar o‘chirib tashlanadi. Keyin ularni qayta tiklab bo‘lmaydi."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"Bu ilova maʼlumotlari butunlay oʻchirib tashlanadi. Unga fayllar, sozlamalar, ma’lumotlar bazalari va boshqa ilovaga tegishli maʼlumotlar kiradi"</string>
     <string name="dlg_ok" msgid="1421350367857960997">"OK"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"Bekor qilish"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-vi/strings.xml b/res/values-vi/strings.xml
index 11f85cbbe8..d3dd173504 100644
--- a/res/values-vi/strings.xml
+++ b/res/values-vi/strings.xml
@@ -2095,7 +2095,7 @@
     <string name="internal_storage" msgid="999496851424448809">"Bộ nhớ trong"</string>
     <string name="recompute_size" msgid="1098091228370999128">"Đang tính toán lại kích thước…"</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"Xóa dữ liệu ứng dụng?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"Tất cả dữ liệu của ứng dụng này sẽ bị xóa vĩnh viễn, bao gồm tất cả các tệp, các tùy chọn cài đặt, tài khoản, cơ sở dữ liệu, v.v."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"Dữ liệu của ứng dụng này sẽ bị xóa vĩnh viễn. Dữ liệu này bao gồm các tệp, chế độ cài đặt, cơ sở dữ liệu và dữ liệu ứng dụng khác."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"OK"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"Hủy"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-zh-rCN/strings.xml b/res/values-zh-rCN/strings.xml
index be2716569d..898845c658 100644
--- a/res/values-zh-rCN/strings.xml
+++ b/res/values-zh-rCN/strings.xml
@@ -2093,7 +2093,7 @@
     <string name="internal_storage" msgid="999496851424448809">"内部存储空间"</string>
     <string name="recompute_size" msgid="1098091228370999128">"正在重新计算大小..."</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"要删除应用数据吗？"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"系统会永久删除此应用的所有数据。删除的内容包括所有文件、设置、帐号、数据库等。"</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"系统将永久删除此应用的数据，其中包括文件、设置、数据库和其他应用数据。"</string>
     <string name="dlg_ok" msgid="1421350367857960997">"确定"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"取消"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-zh-rHK/strings.xml b/res/values-zh-rHK/strings.xml
index 7574669710..1bb9b080b2 100644
--- a/res/values-zh-rHK/strings.xml
+++ b/res/values-zh-rHK/strings.xml
@@ -2096,7 +2096,7 @@
     <string name="internal_storage" msgid="999496851424448809">"內部儲存空間"</string>
     <string name="recompute_size" msgid="1098091228370999128">"正在重新計算大小..."</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"您要刪除應用程式資料嗎？"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"這個應用程式的所有資料都將被永久刪除，包含所有檔案、設定、帳戶、資料庫等。"</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"系統會永久刪除此應用程式的資料，包括檔案、設定、資料庫和其他應用程式資料。"</string>
     <string name="dlg_ok" msgid="1421350367857960997">"確定"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"取消"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-zh-rTW/strings.xml b/res/values-zh-rTW/strings.xml
index 65430dfd66..af7320832b 100644
--- a/res/values-zh-rTW/strings.xml
+++ b/res/values-zh-rTW/strings.xml
@@ -2095,7 +2095,7 @@
     <string name="internal_storage" msgid="999496851424448809">"內部儲存空間"</string>
     <string name="recompute_size" msgid="1098091228370999128">"重新計算大小…"</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"你要刪除應用程式資料嗎？"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"系統會永久刪除這個應用程式的所有資料，包含所有檔案、設定、帳戶、資料庫等。"</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"系統將永久刪除這個應用程式的資料，包括檔案、設定、資料庫和應用程式的其他資料。"</string>
     <string name="dlg_ok" msgid="1421350367857960997">"確定"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"取消"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
diff --git a/res/values-zu/strings.xml b/res/values-zu/strings.xml
index e3605b870d..b74a120a67 100644
--- a/res/values-zu/strings.xml
+++ b/res/values-zu/strings.xml
@@ -2094,7 +2094,7 @@
     <string name="internal_storage" msgid="999496851424448809">"Isitoreji sangaphakathi"</string>
     <string name="recompute_size" msgid="1098091228370999128">"Ilungisa kabusha usayizi..."</string>
     <string name="clear_data_dlg_title" msgid="180446967743732410">"Ngabe ususa yonke imininingo yohlelo lokusebenza?"</string>
-    <string name="clear_data_dlg_text" msgid="7870723948123690332">"Yonke le datha yohlelo lokusebenza izosuswa ngokuphelele. Lokhu kuhlanganisa wonke amafayela, izilungiselelo, ama-akhawunti, izizinda zemininingwane nokunye."</string>
+    <string name="clear_data_dlg_text" msgid="1107610960337399006">"Idatha yale app izosuswa unomphela. Lokhu kuhlanganisa amafayela, amasethingi, isizindalwazi, nenye idatha ye-app."</string>
     <string name="dlg_ok" msgid="1421350367857960997">"KULUNGILE"</string>
     <string name="dlg_cancel" msgid="5164705061530774899">"Khansela"</string>
     <string name="app_not_found_dlg_title" msgid="7122106240981109930"></string>
-- 
2.17.1

